### PR TITLE
2 0 media type

### PR DESCRIPTION
**Pull request by ngilain**
[OO-FEATURE] Media Choice Form Type
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2114
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/249
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/346
https://github.com/open-orchestra/open-orchestra/pull/1149

### CODE REVIEW

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js [hunk 1]
```diff
@@ -8,15 +8,29 @@ import DeleteCheckboxListViewMixin from '../../../Service/DataTable/Mixin/Delete
 class MediaListView extends mix(AbstractDataTableView).with(UrlPaginateViewMixin, DeleteCheckboxListViewMixin)
 {
     /**
+     * Constructor
+     *
+     * @param {Object} options
+     */
+    constructor(options) {
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js]: you can explode options constructor({filterType, selectionMod}) {

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Service/MediaModal/View/MediaModalView.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+     */
+    preinitialize(options) {
+        super.preinitialize(options);
+        this._selectCallback = options.selectCallback;
+        this._filterType = options.filterType;
+        this.events['click #modal-media-choose']  = '_previewMedia';
+        this.events['change #modal-media-format'] = '_updatePreview';
+        this.events['click #modal-media-select']  = '_selectMedia';
+        this.events['click #modal-media-return']  = 'render';
+        Application.getConfiguration().addParameter('mediaViewTemplates', {'image': 'Media/Modal/mediaImageDetailView'});
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Service/MediaModal/View/MediaModalView.js]: why not add this parameter in init of application ?

**Comment by ngilain** [comment]
@alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**